test/e2e/specs/editor/various/block-switcher.spec.js

```diff
@@ -131,9 +132,7 @@ test.describe( 'Block Switcher', () => {
 		await page.keyboard.press( 'ArrowUp' );
 		await page.keyboard.press( 'Enter' );
 		await page.keyboard.type( '## I am a header' );
-		await page.keyboard.down( 'Shift' );
-		await page.keyboard.press( 'ArrowUp' );
-		await page.keyboard.up( 'Shift' );
+		await pageUtils.pressKeys( 'primary+a', { times: 2 } );
```

Comment:
I'm unsure if this is a bug in core or just an e2e test quirk, but when using `pageUtils.pressKeys( 'shift+ArrowUp' )` block toolbar isn't visible.

## Screenshot
![test-failed-1](https://github.com/WordPress/gutenberg/assets/240569/488f3984-ffbb-4314-bbe5-bdbda96e6c32)